selectrum.el

```diff
@@ -1988,11 +1987,15 @@ For PROMPT, COLLECTION, PREDICATE, REQUIRE-MATCH, INITIAL-INPUT,
   (let* ((last-dir nil)
          (msg "Press \\[selectrum-insert-current-candidate] to refresh")
          (sortf nil)
-         (env-completion nil)
+         (is-env-completion nil)
          (coll
           (lambda (input)
             (let* (;; Full path of input dir might include shadowed parts.
                    (path (substitute-in-file-name input))
+                   (is-tramp-path
+                    ;; Check for tramp path, see
+                    ;; `tramp-initial-file-name-regexp'.
+                    (string-match-p "\\`/[^/:]+:[^/:]*:" path))
```

Comment:
FWIW, there is also `file-remote-p`

Reply:
Thanks, I thought this would only catch real remote paths and not `/sudo::` for example I will switch to that.